src/platform/Zephyr/WiFiManager.cpp

```diff
@@ -45,6 +48,25 @@ extern struct wpa_supplicant * wpa_s_0;
 namespace chip {
 namespace DeviceLayer {
 
+namespace {
+
+in6_addr ToZephyrAddr(const Inet::IPAddress & address)
+{
+    in6_addr zephyrAddr;
+
+    static_assert(sizeof(zephyrAddr.s6_addr) == sizeof(address.Addr), "Unexpected address size");
+    memcpy(zephyrAddr.s6_addr, address.Addr, sizeof(address.Addr));
+
+    return zephyrAddr;
+}
+
+net_if * GetInterface(Inet::InterfaceId interfaceId = Inet::InterfaceId::Null())
+{
+    return interfaceId.IsPresent() ? net_if_get_by_index(interfaceId.GetPlatformInterface()) : net_if_get_default();
```

Comment:
I guess `IsPresent()` considers `Inet::InterfaceId::Null()`, no additional checks for NULL iface are needed here?

Reply:
What else could we do if it's null? Return `nullptr` :)? I think only the caller can map it to an appropriate error code etc.

Reply:
Sorry I did not make myself clear. I just mean if the `IsPresent()` returns false if the `Inet::InterfaceId::Null()` object is passed, but it's rather obvious :)

Reply:
Oh, then yes.

Reply:
Done